rest_api/controller/pipeline.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf8
+import shutil
+import uuid
+import os
+from pathlib import Path
+from fastapi import APIRouter, UploadFile, File, HTTPException
+from typing import List
+
+from rest_api.config import PIPELINES_DIR
+from rest_api.controller.utils import PipelineHelper, PipelineSchema
+
+router = APIRouter()
+pipeline_helper = PipelineHelper(PIPELINES_DIR)
+
+
+@router.get("/pipelines", response_model=List[PipelineSchema], response_model_include={'name', 'type', 'status'})
+def get_pipelines():
+    return pipeline_helper.get_pipelines()
+
+
+@router.get('/pipelines/{name}', response_model=List[PipelineSchema], response_model_exclude={'yaml_file'})
```

Comment:
We could have a dedicated endpoint to return the `PipelineSchema`. For instance, `/pipeline/{name}/schema`.

The '/pipelines/{name}' can be used for general information about the Pipeline like the type of pipeline & status(if active).